src/AeroSharp/AeroSharp.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>0.3.1</Version>
+    <Version>0.3.2</Version>
```

Comment:
This should be a minor version bump, since it's adding some new functionality rather than just patching an bug.

Reply:
Got it!  Will adjust - thanks!

Reply:
Bumped!